libs/trustchain/src/sdk.ts

```diff
@@ -383,13 +396,28 @@ async function genericWithJWT<T>(
   });
 }
 
+function encodeToHexaString(value?: Uint8Array, prefix?: boolean): string {
```

Comment:
`crypto.to_hex` is this

Reply:
Yeah It was for testing purpose I'll clean all that